radicle-cli/src/commands/clone.rs

```diff
@@ -133,50 +131,45 @@ pub enum CloneError<H: node::Handle> {
     Payload(#[from] doc::PayloadError),
     #[error("project error: {0}")]
     Project(#[from] ProjectError),
-    #[error("handle error: {0}")]
-    Handle(H::Error),
 }
 
-pub fn clone<G: Signer, H: Handle>(
+pub fn clone<G: Signer>(
     id: Id,
     signer: &G,
     storage: &Storage,
-    node: &mut H,
-) -> Result<(raw::Repository, Doc<Verified>, Project), CloneError<H>> {
+    node: &mut Node,
+) -> Result<(raw::Repository, Doc<Verified>, Project), CloneError> {
     let me = *signer.public_key();
 
-    // Track & fetch project.
-    if node.track_repo(id).map_err(CloneError::Handle)? {
+    // Track.
+    if node.track_repo(id)? {
         term::success!(
             "Tracking relationship restablished for {}",
             term::format::tertiary(id)
         );
     }
 
-    let spinner = term::spinner(format!("Fetching {}..", term::format::tertiary(id)));
-    match node.fetch(id).map_err(CloneError::Handle)? {
-        FetchLookup::Found { seeds, results } => {
-            // TODO: If none of them succeeds, output an error. Otherwise tell the caller
-            // how many succeeded.
-            for result in results.iter().take(seeds.len()) {
-                match &*result {
-                    Ok(_updates) => {}
-                    Err(_err) => {}
-                }
+    // Get seeds.
+    let seeds = node.seeds(id)?;
```

Comment:
Comment probably isn't needed

Reply:
Indeed, I will expand it to something more useful, since lots of stuff is going on in this function.